test/integration/tests/4045-no-run-benchmarks/Main.hs

```diff
@@ -0,0 +1,20 @@
+import Control.Monad (unless,when)
+import Data.List (isInfixOf)
+import StackTest
+
+main :: IO ()
+main = do
+    stack [defaultResolverArg, "clean"]
+    stack [defaultResolverArg, "build"]
+    stackOk ["test"] 
+    stackFail ["bench"] 
+    stackFail ["test", "--no-run-benchmarks"] 
+    putStrLn "DONE"
+
+stackOk :: [String] -> IO ()
+stackOk arg = stackCheckStderr args (\_ -> return ())
+  
+stackFail :: [String] -> IO ()
+stackFail args = stackErrStderr args (\_ -> return ())
+
```

Comment:
Remove the extra two empty lines at the end of this file

Reply:
👍